plugin/plugins/Rules.mk

```diff
@@ -1,6 +1,6 @@
 include mk/header.mk
 
-$(d)_plugins:=$(d)/git $(d)/dagjose $(d)/badgerds $(d)/flatfs $(d)/levelds $(d)/peerlog
+$(d)_plugins:=
```

Comment:
All of the plugins have been removed here due to #8550 (i.e. old xxhash dependencies).

If we upgrade the version of go-libp2p-quic-transport and prometheus/client_golang we should be able to bring back some of the building here as a test mechanism to make sure they can build.